trojsten/special/plugin_prask_7_1_1/views.py

```diff
@@ -0,0 +1,106 @@
+import datetime
+import json
+
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponse, HttpResponseBadRequest
+from django.shortcuts import render
+
+from .constants import LEVELS
+from .mark_level_solved import mark_level_solved
+from .models import UserLevel
+
+
+@login_required()
+def intro(request):
+    return render(
+        request,
+        "plugin_prask_7_1_1/intro.html",
+    )
+
+
+def handle_cookie_level(request, level):
+    out = {"cookies": {}, "variables": {}}
+    if request.COOKIES.get("ukazHeslo") == "true":
+        out["variables"]["password"] = "hihihihi"
```

Comment:
mozno chcme hesla mat v env vars namiesto plaintext v kode vo verejnom repozitari?

Reply:
Jo, to znie rozumne, ale bolo by ich zaroven aj treba vsetky zmenit, lebo inak budu v git historii. To je netrivialne, lebo treba pomenit aj cca vsetky levely.

V tejto serii by som to nechal tak, a v dalsej, ak spravime pokracovanie, to hodime do ENV.

Reply:
Navyse sutaz je pre ZS, pripadne 1. rocnik strednej. Ak niekto najde kod tejto stranky a vyfiltruje z nej hesla tak si zasluzi plny pocet bodov :D

Reply:
Ked squashnes commits pri mergovani, tak sa tie hesla uz nebudu v repo. Ale ak ste okej s tym, tak mne to je jedno.